packages/components/src/tools-panel/tools-panel/hook.ts

```diff
@@ -25,7 +25,7 @@ const generateMenuItems = ( {
 
 	panelItems.forEach( ( { hasValue, isShownByDefault, label } ) => {
 		const group = isShownByDefault ? 'default' : 'optional';
-		menuItems[ group ][ label ] = shouldReset ? false : hasValue();
+		menuItems[ group ]![ label ] = shouldReset ? false : hasValue();
```

Comment:
Not sure why we need to use the `!` here to tell typescript that we are definite that `group` will not be undefined given it is declared just above?